gossip/evmstore/statedb_import.go

```diff
@@ -52,7 +52,7 @@ func (s *Store) ImportLiveWorldState(liveReader io.Reader) error {
 	if err := os.MkdirAll(liveDir, 0700); err != nil {
 		return fmt.Errorf("failed to create carmen dir during FWS import; %v", err)
 	}
-	if err := mptio.ImportLiveDb(mptio.NewLog(), liveDir, liveReader); err != nil {
```

Comment:
I think the decision on how much cache is needed for an import should be left to Carmen in the productive case. So here, the default should be used.

I can see a use case in tests, where you would like to use less. But as I understand, this function is the productive version that is also the one utilized by end-users.